assets/base/i18n/en.json

```diff
@@ -1088,5 +1088,18 @@
   "video.download": "Download video",
   "video.download_description": "This video must be downloaded to play it.",
   "video.failed_description": "An error occurred while trying to play the video.",
-  "your.servers": "Your servers"
+  "your.servers": "Your servers",
+  "channel_info.convert_gm_to_channel": "Convert to a Private Channel",
+  "channel_info.convert_gm_to_channel.screen_title": "Convert to Private Channel",
+  "channel_info.convert_gm_to_channel.warning.header": "Conversation history will be visible to any channel members",
+  "channel_info.convert_gm_to_channel.warning.body": "You are about to convert the Group Message with {memberNames} to a Channel. This cannot be undone.",
+  "channel_info.convert_gm_to_channel.warning.no_teams.header": "Unable to convert to a channel because group members are part of different teams",
+  "channel_info.convert_gm_to_channel.warning.no_teams.body": "Group Message cannot be converted to a channel because members are not a part of the same team. Add all members to a single team to convert this group message to a channel.",
+  "channel_info.convert_gm_to_channel.warning.body.yourself": "yourself",
+  "channel_into.convert_gm_to_channel.team_selector.label": "Team",
+  "channel_into.convert_gm_to_channel.team_selector.placeholder": "Select a Team",
+  "channel_info.convert_gm_to_channel.button_text": "Convert to Private Channel",
+  "channel_info.convert_gm_to_channel.conversion_error": "Something went wrong. Failed to convert Group Message to Private Channel.",
+  "channel_info.convert_gm_to_channel.team_selector_list.title": "Select Team",
+  "generic.back": "Back"
```

Comment:
Did you add these using `i18n-extract`? It feels out of the ordinary that all of them are at the end, instead of alphabetically sorted.

Reply:
I added them manually and hadn't run `i18n-extract` until creation of this PR. I've ran it now and its fixed now.